### ISSUE TITLE
Documentation does not mention the required Python version

### ISSUE THREAD

**Issue by jypeter** [VACUMM/vacumm#15]
@stefraynaud I have just installed the Python2 and Python3 versions of CDAT 8.1, and I have tried to install vacumm in both versions. I just wanted to try, I have not switched to Python3 yet...

- no problem with Python2
- the installer gets stuck when I try to install in Python3. I have tried twice, and ^C the process after 15 mn of guru meditation. If It's not compatible with Python3, is there a way to make the conda install fail faster (with an helpful _not available_ message)?
```
>conda install -n cdatm18_py3 -c conda-forge -c vacumm vacumm
Collecting package metadata: done
Solving environment: \ failed

CondaError: KeyboardInterrupt

```

The documentation does not mention anything about the required python version in the install and prerequisites pages, so you can probably add a note about that

**Comment by stefraynaud** [comment]
Yep, you are right, and it must be mentioned  it in the documentation.
**master** is not python3 compatible.
**clean-imports** is python3 compatible, but it also a fondamental refactory and is no longer based on CDAT (instead on xarray).

**Comment by jypeter** [comment]
Good to know! We'll keep on using the master one for now, since we have not really moved to Python3 yet.

The xarray version should not be a problem, as long as we can keep on installing it in the same conda environment as CDAT. I install a stable CDAT, then clone it and try to install as many extra useful packages as possible in the cloned environment. Let me know what I should type exactly to install this _clean-imports_ and I'll tell you if I can add it to my extended Python3/CDAT

**Comment by johannavarropadron** [comment]
I'ts going to be updated for python 3??
thanks

**Comment by stefraynaud** [comment]
@johannavarropadron yes, and will be available hopefully in a few months.
Note that it will be based on xarray, not CDAT.

**Comment by jypeter** [comment]
@stefraynaud I'm creating (at last) a new python environment based on CDAT 8.2.1 and using Python 3.8

Is there a way to install a non-CDAT based vacumm with python 3.x ? The vacumm web site and github does not mention anything about this

**Comment by epzt** [comment]
which branch should be used for an Python3 install ?
I tried python3 one but not good...
Regards

**Comment by stefraynaud** [comment]
Sorry but vacumm will not be converted to python3.
If you are searching for similar features, please have a look at https://github.com/VACUMM/xoa
It brings a subset of vacumm features and is based on xarray.

**Comment by epzt** [comment]
Thanks for answer. I tried to use a Python script which make use of vacumm functions (create_grid, set_grid, erode_coast), are they present in xoa ?
regards

**Comment by jypeter** [comment]
Thanks @stefraynaud ! I will keep vacumm in my python2 distribution, and install xoa in my latest python3 distribution
